tests/acceptance/features/webUIUpload/uploadEdgecases.feature

```diff
@@ -128,3 +128,19 @@ Feature: File Upload
     #   """
     #   (Any nice error message)
     #   """
+
+  Scenario Outline: Overwrite a file when upload is in progress
```

Comment:
tag the Scenario with the issue and add steps that show the correct behaviour

Reply:
I could not reporduce the issue here, I think this is the correct behavior